integration_tests/src/main/python/delta_lake_delete_test.py

```diff
@@ -176,7 +182,9 @@ def generate_dest_data(spark):
 @pytest.mark.parametrize("partition_columns", [None, ["a"]], ids=idfn)
 @pytest.mark.skipif(is_before_spark_320(), reason="Delta Lake writes are not supported before Spark 3.2.x")
 @datagen_overrides(seed=0, permanent=True, reason='https://github.com/NVIDIA/spark-rapids/issues/9884')
-def test_delta_delete_dataframe_api(spark_tmp_path, use_cdf, partition_columns):
+@pytest.mark.parametrize("enable_deletion_vectors", deletion_vector_values_with_reasons(enabled_xfail_reason="https://github.com/NVIDIA/spark-rapids/issues/12041",
+                                                                                        disabled_xfail_reason="https://github.com/NVIDIA/spark-rapids/issues/12047"), ids=idfn)
```

Comment:
Same question here. Can we have the diabled xfail be specific to databricks 14.3 instead of everywhere?

Reply:
Fixed